tests/bdd/features/volume/resize/resize_offline.feature

```diff
@@ -0,0 +1,46 @@
+Feature: Volume resize
+
+    Background:
+        Given a deployer cluster
+
+    Scenario Outline: expand an unpublished volume
+        Given an unpublished volume with <repl_count> replicas and all are healthy
+        When the volume is expanded
+        Then all the replicas of volume should be resized to new capacity
+        And the volume expand status should be success
+        And the volume is expanded to the new capacity
```

Comment:
try and use imperative rather than declarative in the control-plane bdd's, given these are not k8s specific

```suggestion
        When we issue a volume expand request
        Then the request should succeed
        And the volume is expanded to the new capacity
        And all the volume's replicas should be resized to the new capacity
```

Reply:
I think the order of checks would be better to check that replicas are resized first, then check the volume capacity

```
        And all the replicas of the volume should be resized to the new capacity
        And the volume is expanded to the new capacity
```

Reply:
Sure, whichever way should work